apple_mobileconfig.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"html/template"
 	"net/http"
+	textTemplate "text/template"
```

Comment:
why this?

Reply:
It's noted in the issue, otherwise the xml is escaped